templates/web/README.md.twig

```diff
@@ -1,7 +1,7 @@
 # {{ spec.title }} {{sdk.name}} SDK
 
 ![License](https://img.shields.io/github/license/{{ sdk.gitUserName|url_encode }}/{{ sdk.gitRepoName|url_encode }}.svg?style=flat-square)
-![Version](https://img.shields.io/badge/api%20version-{{ spec.version|url_encode }}-blue.svg?style=flat-square)
+![Version](https://img.shields.io/badge/api%20version-{{ spec.version | split('.') | slice(0,2) | join('.') ~ '.x' | url_encode }}-blue.svg?style=flat-square)
```

Comment:
@Im-Madhur-Gupta Where did this change come from?

Reply:
Hi, facing some medical issues currently, will get back to you in a couple of days.

Reply:
@lohanidamodar Apologies for the delay in reply, I added this code while I was working on the ticket to fix some issue with the version batch but as I can see now we don't need it, hence I have reverted this change.